engine/schema/src/main/resources/META-INF/db/schema-41200to41300.sql

```diff
@@ -540,3 +664,7 @@ CREATE TABLE `cloud`.`template_ovf_properties` (
 -- Add VM snapshot ID on usage helper tables
 ALTER TABLE `cloud_usage`.`usage_vmsnapshot` ADD COLUMN `vm_snapshot_id` BIGINT(20) NULL DEFAULT NULL AFTER `processed`;
 ALTER TABLE `cloud_usage`.`usage_snapshot_on_primary` ADD COLUMN `vm_snapshot_id` BIGINT(20) NULL DEFAULT NULL AFTER `deleted`;
+
+-- Add configuration to set the MTU for OVS and KVM
+INSERT IGNORE INTO `cloud`.`configuration` VALUES ('Network', 'DEFAULT', 'management-server', 'kvm.mtu.size', null,
```

Comment:
This isn't needed if you use the configkey storage of CloudStack

Reply:
changed, please check again